program_info/genicons.sh

```diff
@@ -1,39 +1,73 @@
-#/bin/bash
+#!/bin/bash
```

Comment:
`#!/usr/bin/env bash` might be safer to use.
I do all my scripts like this.

Reply:
As per POSIX `/bin/bash` is basically guaranteed